s3fs/core.py

```diff
@@ -142,7 +169,10 @@ def _ls(self, path, refresh=False):
                     f['Size'] = 0
                     del f['Name']
             else:
-                files = self.s3.list_objects(Bucket=bucket).get('Contents', [])
+                try:
+                    files = self.s3.list_objects(Bucket=bucket).get('Contents', [])
+                except ClientError:
+                    files = []
```

Comment:
Understood, but I still appreciate any review time you can spare for this. Once the writing is merged and other changes like ls above, I don't expect much to change. Will have to make sure, of course, that everything in dask/s3 runs after any merger.